demo/gradio_fastcomposer.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
There is a readme.md under demo folder. Would you please add some simple instructions to the section 3.1.
something like :
run 
python demo/gradio_fastcomposer.py
what to upload and  what is expected to get.

Reply:
ok